portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,104 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>My Portfolio</title>
+
+<link rel="stylesheet" href="bootstrap.min.css">
+<link rel="stylesheet" href="style.css">
+
+<script src="script.js"></script>
+
+<nav class="navbar fixed-top navbar-dark bg-dark">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <span class="navbar-brand mb-0 h1">Prashant Neupane</span>
     </div>
-  </body>
-</html>
+
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="#">HOME</a>
+      <li class="nav-item">
+        <a class="nav-link" href="#about-me">ABOUT</a>
+      <li class="nav-item">
+        <a class="nav-link" href="#connect">CONNECT</a>
+    </ul>
+  </div>
+</nav>
+
+<div id="background-image-0">
+  <div class="quotes">
+    <span class="border">I think therefore I am - Renes Descartes</span>
+  </div>
+</div>
+
+<div id="about-me">
+  <h2>About Me </h2>
+  <p>
+    Hi There, I am Prashant Neupane all the way from Nepal. Nepal is a small
+    landlocked country in the Himalayas. I go to Fisk University in Nashville, TN.
+    I am a Computer Science Major and rising Junior.
+  </p>
```

Comment:
Nit: consider removing optional `</p>`. 

See https://html.spec.whatwg.org/multipage/syntax.html#optional-tags and https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags